README.md

```diff
@@ -273,6 +273,15 @@ Listing asset directories is really slow on Android. You can speed it up though,
 adding `src/android/build-extras.gradle` to the root of your android project (also
 requires cordova-android@4.0.0 or greater).
 
+### Permisson to write to external storage when it's not mounted
+
+Marshmallow requires the apps to ask for permissions when reading/writing to external locations. By default, your app has permission to write to
```

Comment:
Link to [this](http://developer.android.com/guide/topics/data/data-storage.html#filesExternal) maybe?